root/usr/sbin/firewall-migrate

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+set -e
+
+device=$1
+
+bdir=/usr/share/nethserver-firewall-migration-builder
+dest_dir=usr/share/migration
+image=$bdir/nextsecurity.img.gz
+uimage=${image%.gz}
+cimage=$bdir/nextsecurity-custom.img.gz
+url=https://nextsec-testing.fra1.digitaloceanspaces.com/targets/x86/64/nextsecurity-22.03.0-x86-64-generic-ext4-combined-efi.img.gz
+
+avail=$(df -BM --output=avail / | tail -n +2 | cut -dM -f1)
+
+function cleanup()
+{
+    # Cleanup existing loopback devices
+    for lo in $(losetup -a | grep $uimage | awk -F: '{print $1;}')
+    do
+        losetup -d $lo
+    done
+    umount $bdir/mnt 2>/dev/null || true
+    rm -rf $bdir/mnt
+    rm -f $uimage
+}
+
+trap cleanup SIGTERM SIGINT EXIT
+
+if [ "$avail" -lt 400 ]; then
+    echo "Not enough free disk space"
+    exit 1
+fi
+
+if [ ! -f "$image" ]; then
+    echo "Downloading image ... "
+    curl -o $image -L $url
```

Comment:
dest_dir should be created upfront
cannot access /usr/share/nethserver-firewall-migration-builder